src/App.tsx

```diff
@@ -14,11 +14,91 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+interface State {
+  goodItem: string;
+  isGoodSelected: boolean;
+}
+
+class App extends React.Component<{}, State> {
+  state = {
+    goodItem: 'No goods selected',
+    isGoodSelected: false,
+  };
+
+  render() {
+    const { goodItem, isGoodSelected } = this.state;
+
+    const h1Content = isGoodSelected
+      ? `${goodItem} is selected`
+      : `Selected good: - ${goodItem}`;
+
+    let id = 0;
+
+    return (
+      <div className="App">
+        <h1>
+          {goodItem && h1Content}
+
+          <br />
+
+          <button
+            type="button"
+            onClick={() => {
+              this.setState({
+                goodItem: '',
```

Comment:
I think you should have turn to 'No goods selected' for goodItem. Clear removes h1, but in the readme it says `clear the selection`.